.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+      "recommendations": [
+            "golang.go",
+            "adpyke.vscode-sql-formatter"
```

Comment:
let's use this to have uniformly formatted sql queries in the analytics_views folder

Reply:
Ok the problem with this, we need to use pl/sql format or else it breaks some queries (at least it breaks the double $$ sign into two in my latest migration)

Reply:
P.S. in the end I propose to use `inferrinizzard.prettier-sql-vscode` with the postgresql formatted, that handles $$ ok